api/auth/index.js

```diff
@@ -20,9 +20,12 @@ const defaultStrategies = [new LocalStrategy(authenticate)];
 
 module.exports.setup = function setup(
   app,
-  passport = Passport,
-  strategies = defaultStrategies,
-  session = sessionFunction
+  {
+    auth = authenticate,
+    passport = Passport,
+    session = sessionFunction,
+    strategies = defaultStrategies
+  } = {}
```

Comment:
Changed this set of injection parameters into a single object. Makes them more coherent, and also means the order doesn't matter because they're destructured by name. 😄